lock/src/main/java/com/auth0/android/lock/views/ClassicPanelHolder.java

```diff
@@ -320,6 +348,8 @@ public void onClick(View v) {
             event = changePwdForm.submitForm();
         } else if (customFieldsForm != null) {
             event = customFieldsForm.submitForm();
+        } else if (mfaCodeFormView != null) {
```

Comment:
It's not the null check alone is the idea if having test for every type of form we'll ever add to lock and if it's there call it's submit button, in practice it shouldn't be more than one form. Also in the last else we are not performing a null check, probably is not needed but in a 400+ lines class is not straightforward to see if it's always not null.